awx/main/utils/external_logging.py

```diff
@@ -124,3 +127,8 @@ def reconfigure_rsyslog():
             f.write(tmpl + '\n')
         shutil.move(path, '/var/lib/awx/rsyslog/rsyslog.conf')
     supervisor_service_command(command='restart', service='awx-rsyslogd')
+
+
+def send_pg_notify(channel: str, payload: str) -> None:
```

Comment:
is this code needed anymore? I don't see this referenced anywhere

Reply:
it's not. making a PR to remove it